notebooks/pytorch_sparse_quantized_transfer_learning.ipynb

```diff
@@ -0,0 +1,467 @@
+{
```

Comment:
<ul><li>"pre-optimized structure from an already sparse quantized model" to "pruned structure from an already sparse quantized model"</li><li>"<a href="https://neuralmagic.com/blog/pruning-overview/" target="_blank">pruning</a>&nbsp;and&nbsp;<a href="https://arxiv.org/abs/1609.07061" target="_blank">quantization</a> are combined to reduce both the number of parameters and the precision of the remaining parameters to significantly increase the performance of neural networks."</li><li>We should highlight the expected performance increase for this model (~7x depending on the deployment settings)</li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuralmagic/sparseml/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>